packages/components/organisms/f-checkout/src/components/Guest.vue

```diff
@@ -77,18 +78,18 @@ export default {
         * focus on the input field.
         */
         isFirstNameEmpty () {
-            return this.isFieldEmpty('guest', 'firstName');
+            return this.isFieldEmpty(VALIDATIONS.guest, 'firstName');
         },
 
         isLastNameEmpty () {
-            return this.isFieldEmpty('guest', 'lastName');
+            return this.isFieldEmpty(VALIDATIONS.guest, 'lastName');
         },
 
         /*
         * Checks that email field is not empty and is a valid email address.
         */
         isEmailValid () {
-            return !this.isFieldEmpty('guest', 'email') && this.$v.guestValidations.email.email;
+            return !this.isFieldEmpty(VALIDATIONS.guest, 'email') && this.$v[VALIDATIONS.guest].email.email;
```

Comment:
Is `.email.email` correct?

Reply:
The first email is the email field validations.
The second email is the Vuelidate validation for emails

Reply:
Is it worth adding a comment saying this? Not sure it's obvious (as just saw this myself and wondered what it was)